tests/tests_files_elasticsearch_certs_incomplete.yml

```diff
@@ -46,7 +46,9 @@
               outputs: [elasticsearch_output, elasticsearch_output_ops]
         include_role:
           name: linux-system-roles.logging
-        register: result
       rescue:
         - debug:
-            msg: "Caught an expected error - {{ result }}"
+            msg: "Caught an expected error - {{ ansible_failed_result.results }}"
+        - name: assert...
+          assert:
+            that: "'{{ ansible_failed_result.results.0.msg }}' is match('Error: you specified use_cert: true; you must specify all 3 of ca_cert, cert, key, or all 3 of ca_cert_src, cert_src, key_src, or set use_cert: false in the elasticsearch output named elasticsearch_output')"
```

Comment:
This way you check that if the role has failed, it failed with the expected message. But you don't seem to check whether the role has actually failed at all (`rescue` would not be executed).

Reply:
Here's how to check this: https://github.com/pcahyna/storage/blob/787f00cba863a8f87883be6830c1e26488dadd48/tests/tests_disk_errors.yml#L29 .

Reply:
Thanks a lot, @pcahyna. I added your UNREACH check to the test playbooks.843259